experimental/uefi/channel/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "channel"
+version = "0.1.0"
+authors = ["Juliette Pretot <julsh@google.com>"]
+edition = "2021"
+license = "Apache-2.0"
+
+[features]
```

Comment:
As mentioned in the other comment, this crate would not require features if the SerializableRequest is not moved here.